app/managers.py

```diff
@@ -0,0 +1,48 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
```

Comment:
it's better to use .db3 file extension associated with SQLite version 3 databases.